beets/autotag/mb.py

```diff
@@ -447,6 +454,12 @@ def album_info(release):
     if config['musicbrainz']['genres'] and genres:
         info.genre = ';'.join(g['name'] for g in genres)
 
+    # supplementary tags provided by plugins
+    extra_albumdatas = plugins.send('extracting_albumdata', info=release)
```

Comment:
I think `info` might be a somewhat misleading name for the parameter, because it's not a `TrackInfo` or `AlbumInfo` object (as it is for other events). Maybe just `data`?